plugin/src/main/java/com/denizenscript/denizen/scripts/commands/player/FakeSpawnCommand.java

```diff
@@ -0,0 +1,117 @@
+import com.denizenscript.denizencore.scripts.commands.AbstractCommand;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class FakeSpawnCommand extends AbstractCommand {
+
+    public FakeSpawnCommand() {
+        setName("fakespawn");
+        setSyntax("fakespawn [<entity>] [<location>] (players:<player>|...) (d:<duration>{10s})");
+        setRequiredArguments(2, 4);
+    }
+
+    // <--[command]
+    // @Name FakeSpawn
+    // @Syntax fakespawn [<entity>] [<location>] (players:<player>|...) (d:<duration>{10s})
+    // @Required 2
+    // @Maximum 4
+    // @Short Makes the player see a fake entity spawn that didn't actually happen.
+    // @Group player
+    //
+    // @Description
+    // Makes the player see a fake entity spawn that didn't actually happen.
+    // This means that the server will not track the entity,
+    // and players not included in the command will not see the entity.
+    //
+    // You must specify a location and an entity to spawn.
+    //
+    // Optionally, specify a list of players to show the entity to.
+    // If unspecified, will default to the linked player.
+    //
+    // Optionally, specify how long the fake entity should remain for.
+    // If unspecified, will default to 10 seconds.
+    // After the duration is up, the entity will be removed from the player(s).
+    //
+    // @Usage
+    // Use to show a fake creeper in front of the attached player.
+    // - fakespawn creeper <player.forward[5]>
+    //
+    // -->
+
+    @Override
+    public void parseArgs(ScriptEntry scriptEntry) throws InvalidArgumentsException {
+
+        for (Argument arg : scriptEntry.getProcessedArgs()) {
+
+            if (!scriptEntry.hasObject("players")
+                    && arg.matchesPrefix("to", "players")) {
+                scriptEntry.addObject("players", arg.asType(ListTag.class).filter(PlayerTag.class, scriptEntry));
+            }
+            else if (arg.matchesPrefix("d", "duration")
+                    && arg.matchesArgumentType(DurationTag.class)) {
+                scriptEntry.addObject("duration", arg.asType(DurationTag.class));
+            }
+            else if (!scriptEntry.hasObject("entity")
+                    && arg.matchesArgumentType(EntityTag.class)) {
+                scriptEntry.addObject("entity", arg.asType(EntityTag.class));
+            }
+            else if (!scriptEntry.hasObject("location")
+                    && arg.matchesArgumentType(LocationTag.class)) {
+                scriptEntry.addObject("location", arg.asType(LocationTag.class));
+            }
+            else {
+                arg.reportUnhandled();
+            }
+        }
+
+        if (!scriptEntry.hasObject("players") && Utilities.entryHasPlayer(scriptEntry)) {
+            List<PlayerTag> players = new ArrayList<>();
+            players.add(Utilities.getEntryPlayer(scriptEntry));
+            scriptEntry.defaultObject("players", players);

```

Comment:
`scriptEntry.defaultObject("players", Arrays.asList(Utilities.getEntryPlayer(scriptEntry)));`
* I see that this overly long code is copied from ShowFake and will fix that instance of it